src/bot/libraries/games/rps/rps-lobby.ts

```diff
@@ -0,0 +1,143 @@
+import { Message, GuildMember, Guild, TextChannel, DMChannel, GroupDMChannel, Role, MessageReaction } from 'discord.js';
+import { LobbyManager } from '@bot/libraries/games/lobby-manager';
+import { Logger } from '@core/bot/logger';
+import { Lobby } from '@bot/libraries/games/lobby';
+import { Emoji } from '@bot/libraries/emoji';
+import { Client } from 'socket.io';
+
```

Comment:
I don't see any issues with this, I tried to test this command but not really sure what command to use for playing rock papers scissors. Does it involve playing with 2 users or does the bot play with the user?

Reply:
Currently it's only user-to-user. The commands are
"game start" to start a game in that channel
"game join" to join a game already existing in the channel
"game abort" to close a game that's running in that channel
You select the specific game you want to play after using the "game start" command, and the bot will show all response options after entering the command.